wolox-books/src/app/screens/auth/auth.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, Route } from '@angular/router';
+
+import { LocalStorageService } from '../../services/local-storage.service';
+
+@Component({
+  selector: 'app-auth',
+  templateUrl: './auth.component.html',
+  styleUrls: ['./auth.component.scss']
+})
+export class AuthComponent implements OnInit {
+
+  constructor(private localStorageService: LocalStorageService, private router: Router) { }
+
+  ngOnInit() {
+  }
+
+  logOut() {
+    this.localStorageService.removeValue('token');
```

Comment:
instead of using 'token' use the variable declared in the local storage service (line 7) to avoid having a typo by repeating the same text multiple times